src/Libraries/Microsoft.Extensions.AI.Abstractions/Microsoft.Extensions.AI.Abstractions.csproj

```diff
@@ -25,7 +27,7 @@
   </PropertyGroup>
 
   <ItemGroup Condition="'$(TargetFrameworkIdentifier)' != '.NETCoreApp'">
-    <PackageReference Include="System.Text.Json" VersionOverride="$(SystemTextJson8Version)" />
+    <PackageReference Include="System.Text.Json" VersionOverride="$(SystemTextJsonLTSVersion)" />
```

Comment:
And then presumably this is no longer necessary?

Reply:
Yeah it shouldn't be. Removing.